### DISCUSSION TITLE
Does 'yfinance' have no attribute 'pdr_override' anymore?

### DISCUSSION THREAD

**Discussion by peter7041** [ranaroussi/yfinance#2003] [Q&A] [answered]
This code generates an error.
`import pandas as pd
import numpy as np
import yfinance as yf
import datetime as dt
from pandas_datareader import data as pdr

yf.pdr_override()
...`

The error is below:
C:\temp>python ma.py
Traceback (most recent call last):
  File "C:\temp\ma.py", line 7, in <module>
    yf.pdr_override()
    ^^^^^^^^^^^^^^^
AttributeError: module 'yfinance' has no attribute 'pdr_override'

Does 'yfinance' have no attribute 'pdr_override' anymore?

**Comment by ValueRaider** [comment, marked as answer]
See #2001

**Reply by peter7041** [reply]
thank you, ValueRaider.
